### DISCUSSION TITLE
Mods that decrease CS in catch can unintentionally make the map harder

### DISCUSSION THREAD

**Discussion by theredpro-sj** [ppy/osu#30549] [Q&A]
I am unsure if this is an actual issue or an intended feature but mods that lower cs, like easy or difficulty adjust, can make certain osu!catch maps significantly harder. This is because they cause some hyperdashes to be removed and become normal fruit. This results in scenarios where adding easy to a map can cause the star rating to increase dramatically. A likely solution for this issue is to make sure that hyperdash remain when decreasing cs with mods. Below are some examples of maps that are affected when using easy, I have also included links to the map set for ease of use.

https://osu.ppy.sh/beatmapsets/2119567#fruits
<img width="1470" alt="Screenshot 2024-11-08 at 8 45 22 AM" src="https://github.com/user-attachments/assets/006a3643-5b69-42fa-8fde-bf8798c4495e">
https://osu.ppy.sh/beatmapsets/2245593#fruits/4774713
<img width="1470" alt="Screenshot 2024-11-08 at 8 47 10 AM" src="https://github.com/user-attachments/assets/7d70ca5d-7398-4717-b6ba-f1cea1205e75">
https://osu.ppy.sh/beatmapsets/1275097#fruits
<img width="1470" alt="Screenshot 2024-11-08 at 8 48 12 AM" src="https://github.com/user-attachments/assets/d2b71449-ba8b-4b87-b69c-7dbb8f0e21f5">
https://osu.ppy.sh/beatmapsets/1467467#fruits
<img width="1470" alt="Screenshot 2024-11-08 at 8 48 34 AM" src="https://github.com/user-attachments/assets/158ee309-b60e-4260-ada5-515263317aa0">

**Comment by ziv-vy** [comment]
That's a story as old as time.
Yes, it makes it harder. But do you really want to see how Easy mod gives **more** score than nomod?
The mods are simply broken, and have been for decades.
